sdks/python/setup.cfg

```diff
@@ -53,7 +56,11 @@ exclude_lines =
 output = target/site/cobertura/coverage.xml
 
 [isort]
+line_length = 120
```

Comment:
where does this number come from?

Reply:
https://github.com/apache/beam/blob/master/sdks/python/scripts/run_pylint.sh#L105 it's chosen to match isort usage in CI.

Reply:
Came across https://issues.apache.org/jira/browse/BEAM-3745 with touches on isort & lint. perhaps we can close it already or after your changes.